2a_model/src/models/Snakefile_base.smk

```diff
@@ -22,36 +26,11 @@ rule as_run_config:
         asRunConfig(config,output[0])
 
 
-rule prep_io_data:
```

Comment:
So is this moving `prep_io_data` from snakemake to targets, is that right?

Reply:
Right.

Reply:
Awesome, thanks!